dev/ci/scripts/ci-algebra_tactics.sh

```diff
@@ -9,6 +9,8 @@ git_download algebra_tactics
 
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
+export COQEXTRAFLAGS='-native-compiler no'
+
 ( cd "${CI_BUILD_DIR}/algebra_tactics"
   make
   make install
```

Comment:
Perhaps, can we do something like:
```sh
make build
make install
COQEXTRAFLAGS='-native-compiler no' make test-suite
```
to disable the native compute more locally (only for the test suite)? I hope to try it during Rocq'n'share 2025, since we plan to take a look at Nix CI.